### PR TITLE
Actually added userhistory to the app 

### PR DESCRIPTION
**Pull request by echarlotte612**
helps see it then

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -41,11 +41,14 @@ function App() {
...
               </Switch>
             </body>
-            <footer>
-              <Footer />
-            </footer>
+            <div className="app-footer">
+              <footer>
+                <Footer />
+              </footer>
+            </div>
```
Comment (nmunro) [inline frontend/src/App.js:51]: You've got a div outside a body there!

**Review by nmunro — CHANGES_REQUESTED** [review]

File: frontend/src/App.js (hunk 1 above)
Comment (echarlotte612) [reply frontend/src/App.js:51]: maybe that's why the footer isnt playing ball....

Comment (katiemhughes) [reply frontend/src/App.js:51]: That was my fault sorry

Comment (katiemhughes) [reply frontend/src/App.js:51]: That bloody footer

Comment (echarlotte612) [reply frontend/src/App.js]: changed it

**Review by nmunro — CHANGES_REQUESTED** [review]
The <footer><Footer /></footer> need to be before the </body> tag still :)

File: frontend/src/App.js (hunk 1 above)
Comment (nmunro) [reply frontend/src/App.js]: Still needs a bit of a tweak

File: frontend/src/App.js [hunk 2]
```diff
@@ -41,14 +41,15 @@ function App() {
...
                 <Route exact path="/leaderboard" component={Leaderboard} />
+                <Route exact path="/UserHistory" component={UserHistory} />
               </Switch>
             </body>
-            <footer>
-              <Footer />
-            </footer>
+              <footer>
+                <Footer />
+              </footer>
```
Comment (nmunro) [inline frontend/src/App.js:48]: These three lines need to go up and be inside the body tag

**Review by nmunro — CHANGES_REQUESTED** [review]

File: frontend/src/App.js (hunk 2 above)
Comment (echarlotte612) [reply frontend/src/App.js:48]: i've moved them just. before body tag after the switch?

Comment (nmunro) [reply frontend/src/App.js]: I'm not seeing it... It looks on my machine that there's stuff outside the body tags, weirdly...

**Review by nmunro — APPROVED** [review]
Ah, I see it now!

**Pull request merged**